### ISSUE TITLE
Swarmer and Spectre texture need to be reworked

### ISSUE THREAD

**Issue by smolkaguya** [Anuken/Mindustry-Suggestions#1772]
**Describe what you would like changed, and why.**
Swarmer texture should be reworked. It look so weird and kinda out of place. Spectre look like a biotechnical weapon which doesn't fit mindustry theme. Both of these turrets doesn't look robotic. Instead it look fleshy.

**Describe the changes you want to propose. Include possible alternatives.**
 I think spectre texture should be more of battleship gun with the style of meltdown. The turret should have two cannon that is meeting each other instead of the current version which have space between them. As for swarmer, i think it should be more triangle. 
Tldr; spectre look bulky and doesn't look like a thing that will fire projectiles. Swarmer look out of place

**Comment by potatoisagender** [comment]
you can try to draw your own and provide them.

**Comment by Kyllingene** [comment]
![specmelt](https://user-images.githubusercontent.com/69094702/104253257-f887b380-5428-11eb-8d2c-e0635804959e.png)
Are you telling me that the Spectre (left) isn't the same style as the Meltdown (right)? I cannot see how one could consider the Spectre fleshy but the Meltdown fine. If anything, the Spectre with it's tons of sharp angles would be the fine one, and the Meltdown the fleshy one.

**Comment by Humanoid-X** [comment]
@potatoisagender Yeah that's why I hate this type of suggestion.
@Kyllingenee Agree. Imo Spectre is robotic enough and Swarmer look like some sort of "Multi-missle launcher" thing.

**Comment by smolkaguya** [comment]
I'm not saying that spectre isn't same as meltdown. I am saying that the spectre design need to be more slim (it's what i meant woth battleship guns) with meltdown design. Rn it look bulky.

**Comment by Kyllingene** [comment]
> spectre design need to be more slim (it's what i meant woth battleship guns) with meltdown design.

The meltdown is bulkier than the Spectre, especially if you imagine it in 3D. Also, it's a giant death gun that rapidly fires giant bullets. It is not supposed to be "slim."

**Comment by genNAowl** [comment]
Slim isn't even a style that fits **any** of the current Mindustry turrets. If you look at all the turrets, they all are wide, large, guns, that take cover up most of their bases.

**Comment by Kyllingene** [comment]
> Slim isn't even a style that fits any of the current Mindustry turrets.

True. I have made a couple (trash) mods, and my first spriting mistake was making small turrets. They look *horrendous*, even aside from the bad art. Here is an example:
![turrets](https://user-images.githubusercontent.com/69094702/104522024-f4879d00-55b2-11eb-9d58-90de33790924.png)
It may be a little overkill in this pic, but you get the idea. Any ```base size``` : ```turret size``` ratio that is too much more than 1.5 looks terrible.
